_posts/2024-05-28-Anvi’o-visualisations-for-refinement-of-Eukaryotic-MAGs.md

```diff
@@ -0,0 +1,42 @@
+---
+published: true
+category: spotlight
+title: Anvi’o visualisations for refinement of Eukaryotic MAGs
+layout: post
+emg:
+  text: Anvi'o live prototype
+  url: >-
+    http://anvio.mgnify.org
+carousels:
+  - images:
+    - image: /blog/assets/media/images/posts/anvio/SRR12480113_metabat2_3_MERGED.svg 
+    - image: /blog/assets/media/images/posts/anvio/ERR594347_concoct_23_MERGED.svg  
+    - image: /blog/assets/media/images/posts/anvio/ERR594348_concoct_6_MERGED.svg 
+    - image: /blog/assets/media/images/posts/anvio/ERR598996_concoct_100_MERGED.svg
+    - image: /blog/assets/media/images/posts/anvio/ERR599001_concoct_34_MERGED.svg 
+    - image: /blog/assets/media/images/posts/anvio/ERR599023_concoct_42_MERGED.svg
+    - image: /blog/assets/media/images/posts/anvio/ERR599095_concoct_98_MERGED.svg
+    - image: /blog/assets/media/images/posts/anvio/SRR5788032_concoct_67_MERGED.svg
+---
+![Anvi'o logo]({{site.baseurl}}/assets/media/images/posts/anvio/anvio.png){:width="136px"} 
+In the coming months, we will be expanding MGnify Genomes to include eukaryotic genomes. Eukaryotic metagenome-assembled genome (MAG) generation relies on assemblies of metagenomic data, typically performed by metaSPAdes [Nurk *et al.* (2017)](http://www.genome.org/cgi/doi/10.1101/gr.213959.116) or MEGAHIT [Li *et al.* (2015)](https://doi.org/10.1093/bioinformatics/btv033). These tools are not biased toward specific organisms, however compared to prokaryotes, eukaryotic genomes are large, have more compositional variation and contain more repetitive regions, meaning they are complex to both assemble (with many fragments) and to generate MAGs. For example, during binning we have found that a single eukaryotic genome can be split across two MAG bins. To alleviate some of these challenges, EukCC2 [Saary *et al.* (2022)](https://doi.org/10.21203/rs.3.rs-1441815/v1) contains a bin-merging functionality that connects bins from the same taxa, using measures of genome completeness and having paired-end reads connecting the two bins to validate merging. Others have used co-assembly and co-binning of multiple samples to overcome this problem [Delmont *et al.* (2022)](https://doi.org/10.1016/j.xgen.2022.100123), but related samples are not always available, hence MGnify has adopted single-sample approaches. Nevertheless, the challenges posed by eukaryotic MAG generation afford the need for users to inspect the quality of our automatically generated eukaryotic MAGs.
```

Comment:
We need a shorter opener so that the green box on MGnify home page is not too big. E.g.:
```suggestion
In the coming months, we will be expanding MGnify Genomes to include eukaryotic genomes. Take an early look at our plans to enable users to explore eukaryotic genomes' contigs and potential contamination using Anvi'o.

Eukaryotic metagenome-assembled genome (MAG) generation relies on assemblies of metagenomic data, typically performed by metaSPAdes [Nurk *et al.* (2017)](http://www.genome.org/cgi/doi/10.1101/gr.213959.116) or MEGAHIT [Li *et al.* (2015)](https://doi.org/10.1093/bioinformatics/btv033). These tools are not biased toward specific organisms, however compared to prokaryotes, eukaryotic genomes are large, have more compositional variation and contain more repetitive regions, meaning they are complex to both assemble (with many fragments) and to generate MAGs. For example, during binning we have found that a single eukaryotic genome can be split across two MAG bins. To alleviate some of these challenges, EukCC2 [Saary *et al.* (2022)](https://doi.org/10.21203/rs.3.rs-1441815/v1) contains a bin-merging functionality that connects bins from the same taxa, using measures of genome completeness and having paired-end reads connecting the two bins to validate merging. Others have used co-assembly and co-binning of multiple samples to overcome this problem [Delmont *et al.* (2022)](https://doi.org/10.1016/j.xgen.2022.100123), but related samples are not always available, hence MGnify has adopted single-sample approaches. Nevertheless, the challenges posed by eukaryotic MAG generation afford the need for users to inspect the quality of our automatically generated eukaryotic MAGs.
```